CHANGELOG.md

```diff
@@ -5,21 +5,21 @@ This file documents all notable changes in `LogDNA LogSpout Image`. The release
 ## v1.2.0 - Released on May 29, 2019
 
 * Added Tagged Build;
-* Added [semantic versioning](http://semver.org);
-* Added [`CHANGELOG.md`](https://github.com/logdna/logspout/blob/master/CHANGELOG.md);
-* Updated [`LICENSE`](https://github.com/logdna/logspout/blob/master/LICENSE);
+* Added [Semantic Versioning](http://semver.org);
+* Added [CHANGELOG.md](https://github.com/logdna/logspout/blob/master/CHANGELOG.md);
```

Comment:
Actually, this and `LICENSE` below I'm fine leaving the backticks since they're file names